docs/source/getting_started.rst

```diff
@@ -0,0 +1,270 @@
+.. include:: global.rst
+
+Getting Started
+========
+
+This part of the docs will walk you through the main features of remoulade.
+
+Prerequisites
+------
+
+To follow this guide, you will need to set up a new virtual environment and install Remoulade and Requests_::
+
+  $ pip install -U 'remoulade[rabbitmq]' requests
+
+.. _requests: http://docs.python-requests.org
+
+You will also need to install RabbitMQ_, or use Docker_ to run a RabbitMQ image::
+
+  $ docker run -d --name rabbitmq -p 5672:5672 rabbitmq
+
+.. _Docker: https://docs.docker.com/engine/install/ubuntu/
+
+Actors
+------
+
+Let's start by writing a simple function that uses weather-api_ to get the weather for a location and writes it into a file :
+
+.. _weather-api: https://github.com/robertoduessmann/weather-api
+
+.. code-block:: python
+   :caption: get_weather.py
+
+    import requests
+
+
+    def get_weather(city):
+        url = f"https://goweather.herokuapp.com/weather/{city}"
+
+        response = requests.get(url).json()
+
+        file = open('weather', 'a')
+        text = f"{city}:\n{response['temperature']}\n"
+        file.write(text)
+        file.close()
+
+This script simply gets the weather for a city and writes the city and the temperature into a file.
+Running this function gives the expected output::
```

Comment:
I just ran it in command line like for the other examples but didn't explicitly wrote it here :

```
$ python
>>> from get_weather import get_weather
>>> get_weather("Paris")
```


I think it is clearer to execute it each time in command line to avoid changing the code and confusing readers.
I'll change the name of the file though.